website/docs/r/schema_site_anp_epg_bulk_staticport.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "mso"
+page_title: "MSO: mso_schema_site_anp_epg_bulk_staticport"
+sidebar_current: "docs-mso-resource-schema_site_anp_epg_bulk_staticport"
+description: |-
+  Manages MSO Schema Template Application Network Profiles Endpoint Groups Bulk Static Port.
+---
+
+# mso_schema_site_anp_epg_bulk_staticport #
+
+Manages MSO Schema Template Application Network Profiles Endpoint Groups Bulk Static Port.
+
+## Example Usage ##
+
+```hcl
+
+resource "mso_schema_site_anp_epg_bulk_staticport" "static_port" {
+  schema_id            = mso_schema.schema1.id
+  site_id              = mso_schema_site.schema_site.site_id
+  template_name        = "Template1"
+  anp_name             = mso_schema_site_anp_epg.site_anp_epg.anp_name
+  epg_name             = mso_schema_site_anp_epg.site_anp_epg.epg_name
+  static_ports {
+    path_type            = "vpc"
+    deployment_immediacy = "lazy"
+    pod                  = "pod-4"
+    leaf                 = "105"
+    path                 = "eth1/4"
+    vlan                 = 207
+    mode                 = "regular"
+  }
+  static_ports {
+    path_type            = "port"
+    deployment_immediacy = "immediate"
+    pod                  = "pod-1"
+    leaf                 = "102"
+    path                 = "eth1/11"
+    vlan                 = 200
+    micro_seg_vlan       = 3
+    mode                 = "untagged"
+  }
+}
+
+```
+
+## Argument Reference ##
+
+* `schema_id` - (Required) SchemaID under which you want to deploy Static Port.
```

Comment:
I'm not sure if we should use 'you' in the docs. It would be good if we could change this to something like this -
SchemaID under which the Static Port is deployed

Reply:
resolved